lib/src/cupertino/cupertino_popover_menu.dart

```diff
@@ -258,6 +312,16 @@ class RenderPopover extends RenderShiftedBox {
 
     _contentOffset = _computeContentOffset(arrowLength);
 
+    if (_extendAndClipContentOverArrow) {
+      // Compute the size the child wants to be.
+      final desiredSize = child!.getDryLayout(innerConstraints);
+
+      // Add room for the arrow in both top and bottom.
+      innerConstraints = constraints.enforce(
+        BoxConstraints(minHeight: desiredSize.height + arrowLength * 2),
```

Comment:
In the earlier calculation you used `reservedSize`, but here you use `arrowLength * 2`. Is this two different calculations for the same thing? If so, can we stick to one?

Reply:
When `_extendAndClipContentOverArrow` is true, the `reservedSize`'s height is zero, because the content starts at the top of the `RenderObject`.

We can't use the same value.